avatars/animationHelpers.js

```diff
@@ -1225,9 +1228,10 @@ export const _applyAnimation = (avatar, now, moveFactors, timeDiffS) => {
       animationTrackName: k,
       dst,
       isPosition,
+      boneName,
       isFirstBone,
     } = spec;
-
+    
```

Comment:
This looks like VSCode auto formatting.

I know we have a bunch of red squigglies, but can we remove these / paste them back in for the sake of making a really simple PR and then we can do formatting separately?